python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -139,13 +147,140 @@ def _build_fil_classifier(m, data, args, tmpdir):
     bst = xgb.train(params, dtrain, num_rounds)
     bst.save_model(model_path)
 
-    return m.load(
-        model_path,
-        algo=args["fil_algo"],
-        output_class=args["output_class"],
-        threshold=args["threshold"],
-        storage_type=args["storage_type"],
-    )
+    fil_kwargs = {}
+    for param, input_name in (
+        ("algo", "fil_algo"),
+        ("output_class", "output_class"),
+        ("threshold", "threshold"),
+        ("storage_type", "storage_type"),
+        ("precision", "precision"),
+    ):
+        try:
+            fil_kwargs[param] = args[input_name]
+        except KeyError:
+            pass
```

Comment:
This can be expressed much shorter if we can assume that all args are valid fil kwargs:
```suggestion
    arg_mapping = {"fil_algo": "algo"}
    fil_kwargs = {arg_mapping.get(key, key): args[key] for key in args}
```

Otherwise, it's not necessarily shorter, but I would still recommend to use a dict-comprehension:
```suggestion
    arg_mapping = (
        ("algo", "fil_algo"),
        ("output_class", "output_class"),
        ("threshold", "threshold"),
        ("storage_type", "storage_type"),
        ("precision", "precision"),
    )
    fil_kwargs = {
        param: args[input_name]
        for param, input_name in arg_mapping
        if input_name in args
    }
```